src/pages/index.js

```diff
@@ -51,7 +51,7 @@ class Home extends Component {
     return (
       <Layout location={location}>
         <TitleAndMetaTags
-          title="React &ndash; A JavaScript library for building user interfaces"
+          title="React &ndash; User interfaces के निर्माण के लिए एक जावास्क्रिप्ट लाइब्रेरी"
```

Comment:
Any reason you have not translated `user interfaces`?

Reply:
We were not translating ui, so left that as well, or should I?

Reply:
I feel that is only for `UI` since its a shorthand

Reply:
what will be the translation of it?

Reply:
`युज़र इंटरफ़ेस`
How about this.
I checked its hindi meaning. its more complex. How about keeping it like this?

Reply:
interfaces, plural version is the problem 😞

Reply:
`इंटरफेसेस`
How about this?

Reply:
looks confusing to me

Reply:
Then let's keep it `User Interfaces`.
We will come back to this, if it needs any changes

Reply:
I changed it to युज़र इंटरफ़ेसिज़ and committed it along with the other changes you asked for